msr/msfr/core_cfd/3d_fullcore_les/SIZE

```diff
@@ -0,0 +1,42 @@
+c
```

Comment:
this file isnt covered by docs btw

Reply:
what do you mean here? The SIZE is part of the Nek5000 case files, which is included in the PR.

Reply:
the documentation does not describe all the case files?

Reply:
I see now. I am not sure if it is really necessary to cover all case files. The tutorial is more for users with some level of Nek experience, not users with zero experience. The documentation is probably sufficient for whoever knows Nek. I also encouraged users to start with Nek5000 online tutorial first if they are new to Nek.

Reply:
true I remember reading that.
I d still mention the files explicitly in the docs so that someone can search for SIZE and be pointed to a Nek tutorial

Reply:
Thanks for the suggestions. I will make the changes accordingly.